src/components/common/Map/index.js

```diff
@@ -25,6 +33,10 @@ const Map = ({ position }) => {
         url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
       />
       <UpdateCenter position={position} />
+      <NewTarget />
+      {newTargetPos.coords.length != 0 && history.location.pathname == routes.targets.create && (
```

Comment:
Is the pathname check needed? Also, how about extracting `newTargetPos.coords.length` to a new selector with a simpler name like `hasNewTarget`? (probably you can think of a better name 😂 )